sdk/core/azure_core_amqp/README.md

```diff
@@ -6,23 +6,66 @@ NOTE: THIS IS NOT A GENERAL PURPOSE AMQP LIBRARY AND SHOULD NOT BE USED AS SUCH.
 
 This crate is part of a collection of crates: for more information please refer to [https://github.com/azure/azure-sdk-for-rust](https://github.com/azure/azure-sdk-for-rust).
 
-## Example
+## Testing the AMQP Client.
 
-```rust,no_run
-use azure_messaging::*;
+The AMQP package is tested using the standard `cargo test` command line:
 
-#[tokio::main]
-async fn main() -> azure_core::Result<()> {
-    let eventhubs_namespace = std::env::var("AZURE_EVENTHUB_NAMESPACE").expect("missing AZURE_EVENTHUB_NAMESPACE");
+```bash
+cargo test --package azure_core_amqp --all-features
+```
+
+Certain functionality of the test requires that the azure-amqp TestAmqpBroker be running at the time of the test.
+
+To enable this functionality, there are two ways of installing and running the TestAmqpBroker, Scripted and Manual.
+
+### Scripted Broker Install
+For Scripted testing, simply run the powershell script in the sdk/core/azure_core_amqp directory:
+
+```powershell
+PS> .\sdk\core\azure_core_amqp\Test-Setup.ps1
```

Comment:
Don't include the `PS>` prefix. If someone clicks the Copy button baked into many implements' markdown code renderers, it will err.

Relatedly, I recommend separating shell commands in a `bash`, `powershell`, etc. code fence and their output in a `text` fence.